api/src/main/java/org/apache/cloudstack/api/command/admin/network/CreateNetworkOfferingCmd.java

```diff
@@ -241,6 +282,14 @@ public Boolean getForVpc() {
         return forVpc;
     }
 
+    public Boolean getForNsx() {
```

Comment:
Very minor one, but can we name this `isForNsx` and make it return a boolean using the BooleanUtils to make it false when the parameter is null?